### ISSUE TITLE
Terracotta server: Configuration reference

### ISSUE THREAD

**Issue by NiklausHofer** [ehcache/ehcache3#3125]
Hi

The release packages contain a Terracotta server for clustering.

I had the configuration of the service figured out for the 3.8 releases when the configuration file was in XML format. However, it would appear that with the 3.9 release, the format has changed. Now there is a `cluster.cfg` file contained in the release.

Unfortunately I was unable to find documentation for this new format of the configuration file. As of now I am unable to figure out what the values in the file mean and which additional values can be specified.

Due to this, I am not able to upgrade my very simple, single-node cluster as of now.

Could you please point me to the correct documentation?

Thanks and sincerely

Niklaus Hofer

**Comment by mathieucarbou** [comment]
Hello,

I don't know what to answer. Sadly we wrote the doc but for the Enterprise users and as such it is not available in the OSS websites / packaging as of now.

I can redirect you you:

1) Our Terracotta Docker images on Docker Hub which contain some lightweight doc regarding starting servers and images: https://hub.docker.com/u/terracotta

2) Our Enterprise doc: https://documentation.softwareag.com/terracotta/terracotta_10-11/webhelp/index.html#page/terracotta-db-webhelp%2Fco-srv_starting_the_server.html%23

Basically, the `config-tool` was introduced: now the servers can be started and the topology and configuration can be changed at runtime dynamically.
